wire-ios-data-model/Source/Model/Conversation/ZMConversation+MLS.swift

```diff
@@ -49,14 +49,34 @@ extension ZMConversation {
 
     // MARK: Properties
 
-    @NSManaged public var cipherSuite: NSNumber?
+    @NSManaged private var primitiveCiphersuite: NSNumber?
 
     @NSManaged public var epoch: UInt64
 
     @NSManaged public var epochTimestamp: Date?
 
     @NSManaged private var primitiveMessageProtocol: NSNumber
 
+    public var ciphersuite: MLSCipherSuite? {
+        get {
+            willAccessValue(forKey: Self.ciphersuiteKey)
+            let rawValue = primitiveCiphersuite?.int16Value
+            didAccessValue(forKey: Self.ciphersuiteKey)
+
+            guard let rawValue else {
+                return nil
+            }
+
+            return MLSCipherSuite(rawValue: Int(rawValue))
+        }
+
+        set {
+            willChangeValue(forKey: Self.ciphersuiteKey)
+            primitiveCiphersuite = NSNumber(value: newValue?.rawValue ?? 0)
```

Comment:
Is 0 effectively a `nil` cipher suite? Could we be more explicit and store `nil`?

Reply:
Not possible when using scalar values in core data